prod-vars.yml is no longer used, update the docs. (#733)
open-craft/opencraft: documentation/provisioning-sandboxes.md

```

If you pass custom `configuration_extra_settings` to `production_instance_factory`,

> they will override variables from the existing hardcoded settings.

Reviewer: ```suggestion
they will override the variables defined in `instance/models/mixins/openedx_config.py`.
```
